src/climatevision/generator/makeentries.py

```diff
@@ -12,6 +12,13 @@ def make_entries(
     assert year_baseline in YEAR_BASELINE_CHOICES
     assert year_target in YEAR_TARGET_CHOICES
```

Comment:
Evtl. wäre es auch sinnvoll die Tests allg. noch etwas zu erweitern:
z.B. in test_devtool_commands.py Z26: ` ["run", "-ags", "10000000", "-year_ref", "2021", "-year_baseline", "2022", "-year_target", "2040", "-o", filePath],`

Aktuell läuft in den Tests meist nur der Happy Path. Evtl. könnte man noch Tests für die Negativfälle ergänzen.